While generating transcriptomes and getting counts can be a fun and exciting part of RNA-Seq Analysis, we really want to have results to show us what our genes are doing differently between our control and our treatments.

An important part of RNA-Seq Analysis is finding out the names of genes and functional pathways that are associated with them.

This tutorial demonstrates another differential expression package in R, called Gage, that provides some insights into affected pathways using Pathview in R.

###During this session you will learn about:

1. Use of GAGE package to do pathway analysis

   a. "Native workflow"
        
   b. Combined workflow with RNAseq
        
2. Interpret the GAGE output 

3. Use of Pathview to visualize the perturbed KEGG pathways

This tutorial was initially developed by Asela Wijeratne, at Ohio State University as part of the NGS Advanced Bioinformatics Training Course at Kellogg Biological Station, Michigan Statue University, 2015.

The materials have been updated by Adelaide Rhodes, Ph.D. for the Winter 2016 class on Differential Expression Analysis for RNA-Seq.
